src/components/Fruit.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from "react";
+
+function ChooseFruit({text, fruitName, setFruit}){
+
+    function  setFruitMin (){
+        setFruit(fruitName > 0 ? fruitName-1 : fruitName)
+    }
+    function setFruitPlus(){
+        setFruit(fruitName+1);
```

Comment:
Nu zal telkens wanneer `fruitName` veranderd een onnodige rerender van deze functie plaatsvinden. Wat je zou kunnen doen is de huidige waarde gebruiken die je vanuit de setValue functie al krijgt:
```suggestion
        setFruit(currentValue => currentValue +1);
```